examples/event_handling/EventHandling.cpp

```diff
@@ -0,0 +1,224 @@
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+
+#include <vector>
+
+
+namespace
+{
+std::string vec2ToString(const sf::Vector2i& vec2)
+{
+    return '(' + std::to_string(vec2.x) + ", " + std::to_string(vec2.y) + ')';
+};
+} // namespace
+
+
+////////////////////////////////////////////////////////////
+/// \brief Application class
+///
+////////////////////////////////////////////////////////////
+class Application
+{
+public:
+    ////////////////////////////////////////////////////////////
+    Application() :
+    m_window(sf::VideoMode({800u, 600u}), "SFML Event Handling", sf::Style::Titlebar | sf::Style::Close), // Create the main window
+    m_font(sf::Font::openFromFile("resources/tuffy.ttf").value()), // Open the application font
+    m_logText(m_font, "", 20)                                      // Create the log text
+    {
+        m_window.setVerticalSyncEnabled(true);
+        m_logText.setFillColor(sf::Color::White);
+    }
+
+    ////////////////////////////////////////////////////////////
+    void run()
+    {
+        // This example is for demonstration purposes only
+        // All the following forms of event handling have equivalent behavior
+        // In your own code you should decide which form of event handling
+        // suits your needs best and use a single form of event handling
+        auto handlerType = 0;
+
+        while (m_window.isOpen())
+        {
+            if (handlerType == 0)
```

Comment:
Even though it's just a rotating number, I'd still use `enum` values to name the "magic numbers".

Reply:
Fixed.